app_dart/lib/src/request_handlers/github/branch_subscription.dart

```diff
@@ -45,6 +48,11 @@ class GithubBranchWebhookSubscription extends SubscriptionHandler {
     final CreateEvent createEvent = CreateEvent.fromJson(json.decode(webhook.payload) as Map<String, dynamic>);
     await branchService.handleCreateRequest(createEvent);
 
+    final RegExp candidateBranchRegex = RegExp(r'flutter-\d+\.\d+-candidate\.\d+');
+    if (candidateBranchRegex.hasMatch(createEvent.ref!)) {
```

Comment:
@ricardoamador FYI, here is the logic for only creating a commit on candidate branches.